script.js

```diff
@@ -0,0 +1,25 @@
+const menuBtn = document.getElementById('mobile-navi');
+
+const nav = document.querySelector('nav');
+
+const closeBtn = document.getElementById('mobile-close');
+
+const menu = document.getElementsByClassName('menu');
+
+function open() {
+  nav.style.display = 'flex';
+  console.log('its working');
```

Comment:
- [ ] [optional] As part of JS best practice its recommended not to have `console.log`s in your js files